tests/testthat/_snaps/table_egt03.md

```diff
@@ -3,102 +3,102 @@
     Code
       res
     Output
-                                    LOW     NORMAL      HIGH   Missing 
-      —————————————————————————————————————————————————————————————————
-      Placebo (N=86)                                                   
-        n                            0        333        0        3    
-        LOW                          0     84 (25%)      0        0    
-        NORMAL                       0    249 (74.1%)    0        0    
-        HIGH                         0         0         0        0    
-        Missing                      0         0         0     3 (0.9%)
-      Xanomeline High Dose (N=84)                                      
-        n                            0        327        0        1    
-        LOW                          0     82 (25%)      0        0    
-        NORMAL                       0    245 (74.7%)    0        0    
-        HIGH                         0         0         0        0    
-        Missing                      0         0         0     1 (0.3%)
-      Xanomeline Low Dose (N=84)                                       
-        n                            0        335        0        1    
-        LOW                          0     84 (25%)      0        0    
-        NORMAL                       0    251 (74.7%)    0        0    
-        HIGH                         0         0         0        0    
-        Missing                      0         0         0     1 (0.3%)
+                                       LOW         NORMAL     HIGH   Missing 
+      ———————————————————————————————————————————————————————————————————————
+      Placebo (N=86)                                                         
+        n                               37           45        0        2    
```

Comment:
there is a 2 missing somewhere?

Reply:
Hi @shajoezhu,

I looked deeper into the data and it seems like the missing 2 are a result of 2 subjects in the ADEG dataset having `ONTRTFL == NA` for all their records, which means they are included in ADSL (N = 86) but not in ADEG (N = 84) after filtering for `ONTRTFL == "Y"`.

I have resolved the discrepancy by using `adeg_f` for the `N` counts instead of `adsl` so that the counts sum up to the correct totals.